### PR TITLE
Enable disabled question

### PR DESCRIPTION
**Pull request by tfluehmann**
Added the methods to ask the job if it is enabled? or disabled?

### CODE REVIEW

File: lib/sidekiq/cron/job.rb [hunk 1]
```diff
@@ -265,6 +265,16 @@ def enable!
         @status = "enabled"
         save
       end
+      
+      def enabled?
+        return @status == "enabled"
```
Comment (ondrejbartas) [inline lib/sidekiq/cron/job.rb]: why explicit return?

File: test/unit/job_test.rb [hunk 2]
```diff
@@ -149,7 +157,6 @@
       job = Sidekiq::Cron::Job.new('klass' => 'UnknownCronClass')
       assert_equal job.message, {"class"=>"UnknownCronClass", "args"=>[], "queue"=>"default"}
     end
-
```
Comment (ondrejbartas) [inline test/unit/job_test.rb]: please return this space back.

**Comment by ondrejbartas** [comment]
If you fix y comments I will merge it.

**Comment by tfluehmann** [comment]
Done, I am using always the explicit return way, because I like to see always what a method returns.
But if it is best practice in this project I don't mind.
Thx for merging.

File: test/unit/job_test.rb [hunk 3]
```diff
@@ -59,6 +59,14 @@
     it "have destroy method" do
       assert @job.respond_to?(:destroy)
     end
+    
+    it "have save method" do
+      assert @job.respond_to?(:enabled?)
+    end
+    
+    it "have save method" do
```
Comment (ondrejbartas) [inline test/unit/job_test.rb]: wrong naming of test

File: test/unit/job_test.rb [hunk 4]
```diff
@@ -59,6 +59,14 @@
     it "have destroy method" do
       assert @job.respond_to?(:destroy)
     end
+    
+    it "have save method" do
```
Comment (ondrejbartas) [inline test/unit/job_test.rb]: wrong naming of test

File: lib/sidekiq/cron/job.rb [hunk 5]
```diff
@@ -265,6 +265,16 @@ def enable!
...
+      
+      def enabled?
+        @status == "enabled"
+      end
+      
+      def disabled?
+        !enabled?
+      end
+      
+
```
Comment (ondrejbartas) [inline lib/sidekiq/cron/job.rb]: too many whitespaces

**Comment by tfluehmann** [comment]
How is it now?

**Pull request merged**